namespaces/api-description-namespace.md

```diff
@@ -458,7 +472,23 @@ HTTP request message.
 - `element`: httpRequest (string, fixed)
 - `attributes`
     - `method` (string) - HTTP request method. The method value SHOULD be inherited from a parent transition if it is unset.
-    - `href` (Href) - A concrete URI for the request. The href SHOULD be inherited from a parent transition by expanding the href and hrefVariables if unset.
+    - `href` (Templated Href) - URI Template for this HTTP request.
+
+        If present, the value of the `href` attribute SHOULD be used to resolve
+        target URI of the http request.
```

Comment:
I think the following part doesn't flow very well:

> used to resolve target URI

Perhaps the following would sound better?

```diff
- be used to resolve target URI of the http request.
+ be used to resolve the target URI of the HTTP request.
```